packages/bridge/bridge-react/src/router-v6.tsx

```diff
@@ -60,7 +60,7 @@ function WraperRouterProvider(
     return <RouterProvider router={MemeoryRouterInstance} />;
   } else {
     const BrowserRouterInstance = createBrowserRouter(routers, {
-      basename: routerContextProps.basename,
+      basename: routerContextProps.basename || router.basename,
```

Comment:
how about fix to this:  `basename: routerContextProps.basename || router?.basename`

Reply:
i think there is no need to add `?.` ， this `router` prop will always exist, otherwise ts will throw error

Reply:
ok